docs/faq/troubleshooting/error-line-endings.md

```diff
@@ -2,9 +2,9 @@
 
 Codacy executes the `git diff` command when analyzing new commits and pull requests to identify the lines of code that were changed. Codacy then uses this information to display the issues that were caused by the changes introduced by the commits or pull requests.
 
-If you have files in your repository that use the carriage return (CR) as the line end control character, the command `git diff` doesn't correctly identify line endings in the changed files. Because of this, Codacy is unable to use the output of the command and the **Diff** step of your commit or pull request analysis logs will display the message `An error occurred during this step. Please, retry your analysis or contact support`.
+If you have files in your repository that use the carriage return (CR) as the line end control character, the command `git diff` doesn't correctly identify line endings in the changed files. Because of this, Codacy is unable to use the output of the command and the **Diff** step of your commit or pull request analysis logs will display the message `An error occurred during this step. Please, retry your analysis or contact support`.<!-- TODO ALA-643 Confirm message is still valid -->
```

Comment:
Is this still valid?

If so, I'll need some help reproducing this issue to get the screenshot.

Reply:
I tried manually adding a CR. Is this it?

<img width="653" alt="image" src="https://github.com/codacy/docs/assets/11616750/a8c87def-716c-42cb-8886-6bdd3c0d263c">